### PR TITLE
Add pluginRepositories tag extraction

### PR DESCRIPTION
**Pull request by giorgioprevitera**

### CODE REVIEW

File: parser_test.go [hunk 1]
```diff
@@ -240,4 +248,16 @@ func TestUnmarshal(t *testing.T) {
 	if len(project.Dependencies) != 8 {
 		t.Errorf("expecting 8 dependencies found %d", len(project.Dependencies))
 	}
+
+	if project.PluginRepositories[0].Id != "private-plugin-repository" {
+		t.Errorf("pluginRepository[0] id does not match (expected: private-plugin-repository, found: %s)", project.PluginRepositories[0].Id)
+	}
+
+	if project.PluginRepositories[0].Name != "My private plugin repository" {
+		t.Errorf("pluginRepository[0] id does not match (expected: My private plugin repository, found: %s)", project.PluginRepositories[0].Name)
```
Comment (creekorful) [inline parser_test.go]: name instead of id

File: parser_test.go [hunk 2]
```diff
@@ -240,4 +248,16 @@ func TestUnmarshal(t *testing.T) {
...
+	}
+
+	if project.PluginRepositories[0].Url != "http://localhost:8081/repository/maven-private/" {
+		t.Errorf("pluginRepository[0] id does not match (expected: http://localhost:8081/repository/maven-private/, found: %s)", project.PluginRepositories[0].Url)
```
Comment (creekorful) [inline parser_test.go]: url instead of id

**Review by creekorful — CHANGES_REQUESTED** [review]
Hello! 
Thank you for your PR :)

There's a little typo in the test case, do you mind fixing it?

File: parser_test.go (hunk 1 above)
Comment (giorgioprevitera) [reply parser_test.go]: done

File: parser_test.go (hunk 2 above)
Comment (giorgioprevitera) [reply parser_test.go]: done

**Review by creekorful — APPROVED** [review]

**Comment by creekorful** [comment]
Thanks for you contribution! 😄

**Pull request merged**